ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetDetailedObjects.java

```diff
@@ -62,55 +68,120 @@ public CliCommand init(final Arguments args) throws Exception {
         processCommandOptions(EMPTY_LIST, optionalArgs, args);
 
         this.filterParams = args.getFilterParams();
+        checkFilterParams();
         this.bucketName = args.getBucket();
 
-        // not supported in current version
-        this.prefix = null;
+        this.prefix = null; // doesn't work
         return this;
     }
 
+    private boolean checkFilterParams() throws BadArgumentException {
```

Comment:
This method always returns true or throws an exception when it fails.  It should either return void and throw the exception, or return true/false.